api/users/serializers.py

```diff
@@ -373,6 +367,7 @@ class UserSettingsSerializer(JSONAPISerializer):
     two_factor_enabled = ser.SerializerMethodField()
     subscribe_osf_general_email = ser.SerializerMethodField()
     subscribe_osf_help_email = ser.SerializerMethodField()
+    deactivation_requested = ser.BooleanField(source='requested_deactivation', required=False)
```

Comment:
This was the field name requested in the ticket - field on User is `requested_deactivation`.